src/ansi-c/c_typecheck_code.cpp

```diff
@@ -11,6 +11,8 @@ Author: Daniel Kroening, kroening@kroening.com
 
 #include "c_typecheck_base.h"
 
+#include <ansi-c/c_expr.h>
```

Comment:
Move this further down, underneath `#include "ansi_c_declaration.h"`, and use `#include "c_expr.h"` for consistency.

Reply:
fixed